Update Astro rooting page for firmware V07, adding A/B slot instructions

ZimbiX opened the pull request:
I've added the new boot images, and annotated MD5 hashes for all of them. I'm now appending the Magisk version to the filename, as there was an update since the first image was produced.

Adding instructions for determining which A/B slot's in use is now necessary, since it's no longer always going to be A.

Resolves https://github.com/shymega/planet-devices-wiki-prs/issues/17.

ZimbiX committed:
Update Astro rooting page for firmware V07, adding A/B slot instructions

I've added the new boot images, and annotated MD5 hashes for all of them. I'm now appending the Magisk version to the filename, as there was an update since the first image was produced.

Adding instructions for determining which A/B slot's in use is now necessary, since it's no longer always going to be A.

shymega commented:
This PR looks good, but I'm _really_ reluctant to host boot images on Mega.nz. I would rather we linked to files (or even a release with the boot images) on the `planet-devices` repository. I just can't be assured that Mega.nz, or other third-party hosting (other than S3, for firmware files that I plan to upload there, and LineageOS images) won't be removed, tampered with, or otherwise made inaccessible.

ZimbiX commented:
Sure. Go ahead and copy them there and update the links

shymega committed:
Change links to use GitHub instead of Mega.nz

These files are published in
[shymega/planet-devices][https://github.com/shymega/planet-devices].

shymega commented:
Rebased against mainline, changed links. Let me know if you're good to merge.

ZimbiX commented:
Cheers. LGTM. Though for cleaner URLs, I wouldn't reference the commit sha; but up to you.

shymega commented:
> Cheers. LGTM. Though for cleaner URLs, I wouldn't reference the commit sha; but up to you.

I did that deliberately. It's a deterministic way of ensuring each boot.img is at a set point in time.
